pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -177,6 +180,14 @@ void updateSegmentMetrics() {
           _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.PERCENT_SEGMENTS_AVAILABLE, 100);
           continue;
         }
+
+        if (!idealState.isEnabled()) {
+          LOGGER.warn("Table {} is disabled. Skipping segment status checks", tableName);
```

Comment:
Not sure if it is good to put out a warning here every 5 minutes.

Reply:
Good point. I've updated the code to not log it more than once a day. Let me know if this works better.